<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tool_Launcher_UI</class>
 <widget class="Tool_Launcher_UI" name="Tool_Launcher_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Image Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Image_Folder_Input"/>
     </item>
     <item>
      <widget class="QPushButton" name="Find_File_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="Run_Tool_Button">
     <property name="text">
      <string>Run Tool </string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tool_Launcher_UI</class>
   <extends>QWidget</extends>
   <header>Tool_Launcher_UI</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
